docs/update.js

```diff
@@ -63,39 +65,49 @@ function getUpdatedValue(toggle) {
 
 /**
  * Set in setUpInitialToggleValues.
- * @type {boolean}
+ * @type {ScoreParameters}
  */
-let hasDamageAsset;
+let scoreAssetCreationParameters;
 
 /**
  * Initializes damage-related toggle values based on whether or not we have
  * a damage asset.
- * @param {Promise<Object>} disasterMetadataPromise
+ * @param {Promise<DisasterDocument>} disasterMetadataPromise
  * @param {google.maps.Map} map
  * @return {Promise<Object>} returns all the toggle initial values.
  */
-function setUpToggles(disasterMetadataPromise, map) {
-  return disasterMetadataPromise.then((doc) => {
-    hasDamageAsset = !!doc.data().assetData.damageAssetPath;
-    if (hasDamageAsset) {
-      toggles.set(damageThresholdKey, 0.5);
-      toggles.set(povertyWeightKey, 0.5);
-    }
-    createToggles(map);
-    return getToggleValues();
-  });
+async function setUpScoreComputationParameters(disasterMetadataPromise, map) {
+  ({scoreAssetCreationParameters} = await disasterMetadataPromise);
+  if (!!scoreAssetCreationParameters.damageAssetPath) {
+    toggles.set(damageThresholdKey, 0.5);
```

Comment:
@ruthtalbot  was wondering if we should set these lower by default, for the case where a journalist looks at it. 0.5 is pretty high threshold and it might make nothing show up at first glance.

Reply:
Will leave that for a separate PR